Update README.md (#1068)
containers/skopeo: README.md

Please read the [contribution guide](CONTRIBUTING.md) if you want to collaborate in the project.

## Commands
| Command                                            | Description                                                                                  |
| -------------------------------------------------- | ---------------------------------------------------------------------------------------------|
| [skopeo-copy(1)](/docs/skopeo-copy.1.md)           | Copy an image (manifest, filesystem layers, signatures) from one location to another.        |

> | [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark image-name for deletion.                                                                |

Reviewer: Side note, the skopeo-delete(1) page shows a `--force` option in the examples section, but it is not listed as an option in the options section.
Maybe:
```suggestion
| [skopeo-delete(1)](/docs/skopeo-delete.1.md)       | Mark the image-name for later deletion by the registry's garbage collector.                                                                |
```

Reviewer 2: The primary user-relevant outcome is that the image may be _immediately_ impossible to pull. GC is a secondary internal implementation details only server administrators care about.

Reviewer: FWIW, I picked this and the other point based on info on the respective man pages.  Perhaps they need embellishing too?

> | [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry                                |

Reviewer: ```suggestion
| [skopeo-inspect(1)](/docs/skopeo-inspect.1.md)     | Return  low-level  information about image-name in a registry.                                |
```

> | [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags the transport-specific image repository.                               |

Reviewer: ```suggestion
| [skopeo-list-tags(1)](/docs/skopeo-list-tags.1.md) | Return a list of tags for the transport-specific image repository.                               |
```

| [skopeo-login(1)](/docs/skopeo-login.1.md)         | Login to a container registry.                                                               |
| [skopeo-logout(1)](/docs/skopeo-logout.1.md)       | Logout of a container registry.                                                              |

> | [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest of manifest-file and write it to standard output.   |

Reviewer: I must have missed this when it came in for a review, and I see skopeo-manifest-digest(1) has the same line in it's title.  Does this make more sense?
```suggestion
| [skopeo-manifest-digest(1)](/docs/skopeo-manifest-digest.1.md)    | Compute a manifest digest for a manifest-file and write it to standard output.   |
```

> | [skopeo-standalone-sign(1)](/docs/skopeo-standalone-sign.1.md)    | Simple sign an image.                                                         |

Reviewer: ```suggestion
| [skopeo-standalone-sign(1)](/docs/skopeo-standalone-sign.1.md)    | Debugging tool - Publish and sign an image in one step.                                                         |
```

Reviewer 2: That’s not what the command does; it only creates a local file from local files with no registry interaction at all.

> | [skopeo-standalone-verify(1)](/docs/skopeo-standalone-verify.1.md)| Verify an image signature.                                                    |

Reviewer 2: I’d prefer to not promote the three above much. (OTOH they are not _completely hidden_ like `skopeo layers`.)

(Would it make sense to have this above the examples, or combine it with the “ Skopeo can perform operations which consist of:” list? That can clearly be deferred to a different PR.)

Author: If we want to deprecate the use of these, we should hide them from users, and stop displaying them from skopeo --help.

But lets get this merged and then we can discuss removing the visibility of these commands
